client/src/data/reducers/index.ts

```diff
@@ -6,10 +6,12 @@
 import { combineReducers } from 'redux';
```

Comment:
Could we undo all the changes to redux files?
+ client/src/data/reducers/index.ts
+ client/src/data/reducers/requestGroupReducer.ts
+ client/src/data/store.ts
+ client/src/data/actions/requestGroupAction.ts
+ client/src/data/actions/requestGroupsActions.ts
+ client/src/data/actionTypes.ts

Reply:
@kevinwang2040 can u take a look at this

Reply:
Removed them.